dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/Ds3PanelPresenter.java

```diff
@@ -367,7 +367,7 @@ private void createTabAndSetBehaviour(final Session newSession) {
     private ImmutableList<Ds3JobTask> getActiveItemsInSession() {
         return jobWorkers.getTasks().stream()
                 .filter(ds3JobTask ->getSession().containsTask(ds3JobTask))
-                .filter(task -> task.isInCache())
+                .filter(task -> !task.isInCache())
```

Comment:
why are we changing the filter to the opposite of what it was?

Reply:
It was named confusingly to the point I kept thinking it was wrong, so now it's consistently named.